src/python/kiss_icp/datasets/rosbag.py

```diff
@@ -69,12 +69,13 @@ def read_point_cloud(self, bagfile: Path, topic: str, idx: int):
 
         t_field = None
         for field in msg.fields:
-            if field.name == "t" or field.name == "timestamp":
+            if field.name == "t" or field.name == "timestamp" or field.name == "time":
                 t_field = field.name
         timestamps = np.ones(points.shape[0])
         if t_field:
             timestamps = np.array(list(self.pc2.read_points(msg, field_names=t_field)))
-            timestamps = timestamps / np.max(timestamps)
```

Comment:
I'm more into one-liners for these things, but it's personal taste
```py
timestamps = timestamps / np.max(timestamps) if t_field != "time" else timestamps
```

Reply:
Done c297133565d7aeb557b1a9d549ded93747722ef1